extensions/interactions/MultipleChoiceInput/protractor.js

```diff
@@ -36,13 +36,21 @@ var customizeInteraction = function(elem, richTextInstructionsArray) {
 // the options.
 var expectInteractionDetailsToMatch = function(
     elem, richTextInstructionsArray) {
-  elem.all(by.repeater('choice in $ctrl.choices track by $index'))
+  elem.all(by.repeater('choice in $ctrl.choices'))
     .then(function(optionElements) {
+      var choices = [];
       expect(optionElements.length).toEqual(richTextInstructionsArray.length);
       for (var i = 0; i < optionElements.length; i++) {
-        forms.expectRichText(optionElements[i].element(by.css(
+        optionElements[i].element(by.css(
           '.protractor-test-multiple-choice-option'
-        ))).toMatch(richTextInstructionsArray[i]);
+        )).getText().then(function(text) {
+          choices.push(text);
+          if (choices.length === 4) {
+            for (var j = 0;j < choices.length; j++) {
```

Comment:
Missing space after semicolon. Did the linter not catch this?

Reply:
I think the linter didn't catch this. Done.